### PR TITLE
fix: broken list view pagination

### LINKED ISSUE #21789: Content Manager pagination is broken as of 5.0.3
### Required System information

- Node.js version: v20
- NPM version: yarn v1
- Strapi version: 5.0.4 and 5.0.3
- Database: SQLite
- Operating system: Linux Mint 21.2
- Is your project Javascript or Typescript: Javascript

### Describe the bug

When paginating within the content-manager, changes pages yields an invalid admin REST Query:

`http://localhost:1337/content-manager/collection-types/api::article.article?[object%20Object]?page=2&pageSize=10&sort=title%3AASC`

Specifically the problem is here: `[object%20Object]?page=2`

### Steps to reproduce the behavior

1. Create a content-type with any field
2. Create more than 10 entries
3. Try to paginate

### Expected behavior

Should paginate properly :) 

### Screenshots

Reproduction video:

https://www.loom.com/share/c5a773d26cb74308b920e6df0235a9c7?sid=50da458d-2463-4147-a048-661efeb8eab4

### Code snippets

N/A

### Additional context

N/A

### PR DESCRIPTION
**Pull request by remidej**
### What does it do?

Fixes the way query params are written into the URL of the list

It seems like when passing a params object in the config, RTK query stringifies it properly, but here an object was put directly into the string.

### How to test it?

Have 2+ pages of content. Navigate between pages. Then content should reflect the page you're on
### Related issue(s)/PR(s)

fixes #21789
introduced in #21406

### CODE REVIEW

**Review by alexandrebodin — COMMENTED** [review]
so this was borken way before 5.0.3 ? 

Can you add some test on RTK ?

**Comment by remidej** [comment]
According to our release notes the would have been released in 5.0.2

**Comment by oiorain** [comment]
QA LGTM.
Nice job! ⚡

**Comment by jhoward1994** [comment]
Thanks this looks good, a test to cover it would be good. Maybe `tests/e2e/tests/content-manager/listview.spec.ts` is the place to add it?

**Comment by remidej** [comment]
@jhoward1994 completely agree, I'm working on adding a pagination e2e test to this PR

File: packages/core/content-manager/admin/src/services/documents.ts [hunk 1]
```diff
@@ -166,7 +166,7 @@ const documentApi = contentManagerApi.injectEndpoints({
       }
     >({
       query: ({ model, params }) => ({
-        url: `/content-manager/collection-types/${model}${params ? `?${params}` : ''}`,
+        url: `/content-manager/collection-types/${model}`,
```
Comment (alexandrebodin) [inline packages/core/content-manager/admin/src/services/documents.ts:169]: Can you confirm the original fix still works ?

Comment (remidej) [reply packages/core/content-manager/admin/src/services/documents.ts:169]: yes i tested this manually

**Comment by oiorain** [comment]
QA LGTM

**Review by alexandrebodin — APPROVED** [review]
code LGTM 👍

**Comment by neilsmith** [comment]
Just in case it's helpful: as well as breaking pagination, I believe the same issue also broke searching for multiple words (i.e. searching for "foo" would find a record "foo bar baz" - but searching for "foo bar" would yield no results.)

This fix also seems to resolve that issue!

Don't know if that would be worth covering in an e2e test too?

**Pull request merged**